rpm/qtdeclarative.spec

```diff
@@ -268,6 +285,14 @@ touch .git
 
 make %{?_smp_mflags}
 
+%if 0%{?qt5_release_build}
```

Comment:
Thanks, added this test here. Is there a standard way to set this flag to build debug vs release with the spec file, or is it a magical flag set from OBS when it builds?